### PR TITLE
Add a RPM vs frequency spectrum plot

### PR DESCRIPTION
**Pull request by tbolin**
Add a RPM vs frequency spectrum plot. The plot works essentially the same way as the throttle vs frequency plot but with the motor rates in Hz instead of a percentage of rcCommands[3].

for_github

### CODE REVIEW

File: js/graph_spectrum_calc.js
Comment (haslinghuis) [inline js/graph_spectrum_calc.js:76]: Please avoid var. Use let or const instead to limit scope.

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Please fix sonar issues.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Pull request merged**
